### ISSUE TITLE
Fix known accessibility issues

### ISSUE THREAD

**Issue by lgactna** [lgactna/ncc-web#22]
There are a variety of accessibility issues on the site, including:
- lack of an alt attribute for image-only fields
- various broken CSS things, like skipping headers
- other things I probably don't know of in WCAG 2

These should all be fixed as we approach maturity.
